packages/web-components/src/components/masthead/masthead-button-cta.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { customElement } from 'lit-element';
+import ddsSettings from '../../internal/vendor/@carbon/ibmdotcom-utilities/utilities/settings/settings';
+import { CTA_TYPE } from '../cta/defs';
+import DDSButtonExpressive from '../button/button';
+import styles from './masthead.scss';
+
+export { CTA_TYPE };
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Masthead Button CTA.
+ *
+ * @element dds--masthead-button-cta
+ */
+@customElement(`${ddsPrefix}-masthead-button-cta`)
+class DDSMastheadButtonCTA extends DDSButtonExpressive {
+  static styles = styles;
+}
+
+export default DDSMastheadButtonCTA;
```

Comment:
This component exists in this PR (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/9275/files#diff-ea9df841ca5469043d16d6ce42b324a0e48da3224276aca690c5b3147716dd4f). There might be a potential merge conflict.